### PR TITLE
adding code block examples

### PR DESCRIPTION
**Pull request by runleonarun**
## What are you changing in this pull request and why?
Adding code block examples per and internal Slack conversation.

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: contributing/content-style-guide.md [hunk 1]
```diff
@@ -120,6 +120,8 @@ Use [code blocks](#code-blocks) for longer or more complex commands. Do _not_ us
 
 Keep lines in code samples to about 60 characters, to avoid requiring readers to scroll horizontally in the code block. Locate explanatory text before the code block, rather than using comments inside the code block.
 
+You can look at the different [languages you can use in code blocks](https://markdown.land/markdown-code-block#markdown-code-block-language-list), which will change how the code example is formatted. For example ` ```yaml` or ` ```shell` are commonly used in the dbt product documentation.
```
Comment (nghi-ly) [inline contributing/content-style-guide.md]:
```suggestion
You can look at the different [languages you can use in code blocks](https://markdown.land/markdown-code-block#markdown-code-block-language-list), which will change how the code example is formatted (like highlighting). For example, `yaml` or `shell` are commonly used in the dbt product documentation. 
```

**Review by nghi-ly — COMMENTED** [review]
thank you! left one sugg

**Comment by mirnawong1** [comment]
hey @runleonarun am i ok committing this suggestion and merging for you?

**Comment by mirnawong1** [comment]
hey @runleonarun - following up on this, i'll go ahead and commit @nghi-ly 's suggestion and merge this for you - hope that's ok! if not, i can revert this for you.

**Review by mirnawong1 — APPROVED** [review]

**Pull request merged**